frontend/app/(dashboard)/people/[id]/page.tsx

```
@@ -629,15 +630,21 @@ const sharesColumns = [
+  }
 
-  return shareHoldings.length > 0 ? (
-    <DataTable table={table} />
-  ) : (
-    <Placeholder icon={CircleCheck}>This investor does not hold any shares.</Placeholder>
-  );
+  if (!shareHoldings || shareHoldings.length === 0) {
+    return <Placeholder icon={CircleCheck}>This investor does not hold any shares.</Placeholder>;
+  }
+
+  const table = useTable({ data: shareHoldings, columns: sharesColumns });
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)